src/emu/debug/debugcpu.cpp

```diff
@@ -1008,6 +1007,37 @@ u64 debugger_cpu::expression_read_memory(void *param, const char *name, expressi
 			break;
 		}
 
+		case EXPSPACE_OPCODE:
+		{
+			device_t *device = nullptr;
+			device_memory_interface *memory;
+
+			if (name != nullptr)
+				device = expression_get_device(name);
+			if (device == nullptr || !device->interface(memory))
+			{
+				device = get_visible_cpu();
+				memory = &device->memory();
+			}
+			if (memory->has_space(AS_DECRYPTED_OPCODES + (spacenum - EXPSPACE_OPCODE)))
```

Comment:
I'll redo this again. I did it in a way that meant that I don't have to check 5188 occurrences of the .o in the cheat collection to see which ones are valid and which ones worked because it was treating .o the same as .m or .r.

Don't think I'm going to get out of doing all the work on the cheat collection unfortunately...so I best knuckle down before I do another commit to get the cheat collection ready.